acceptance-tests/src/test/scala/ch/datascience/graph/acceptancetests/knowledgegraph/LineageQuerySpec.scala

```diff
@@ -97,6 +97,7 @@ class LineageQuerySpec extends FeatureSpec with GivenWhenThen with GraphServices
       lineage(projectPath: "namespace/lineage-project", filePath: "figs/grid_plot.png") {
         nodes {
           id
+          location
```

Comment:
you're already implementing https://github.com/SwissDataScienceCenter/renku/issues/696? :)

Reply:
:) It's just a relative path inside a project structure.

Reply:
I see that now, I got too excited. :)

Reply:
:)